platform/ios/src/MGLMapboxEvents.m

```diff
@@ -245,16 +249,28 @@ - (void)userDefaultsDidChange:(NSNotification *)notification {
     });
 }
 
+- (BOOL)isLowPowerModeEnabled {
```

Comment:
This isn’t a property, but it is named after the getter for a property — should we name it differently?